newrelic-agent/src/main/java/com/newrelic/agent/config/AgentConfigImpl.java

```diff
@@ -356,7 +356,7 @@ private String initSSLConfig() {
             if (caBundlePath != null) {
                 Agent.LOG.log(Level.INFO, "use_private_ssl configuration setting has been removed.");
             } else {
-                Agent.LOG.log(Level.SEVERE, "use_private_ssl configuration setting has been removed. Please use ca_bundle_path instead.");
+                Agent.LOG.log(Level.SEVERE, "The use_private_ssl configuration setting has been removed and will be ignored. The agent will use the JVM/JRE truststore by default unless you configure ca_bundle_path to use a different truststore.\n");
```

Comment:
Is the newline character (`\n`) needed here or is it just a copy & paste artifact? Let's remove it if it isn't intentional.